packages/core/src/mpi/get-patient-by-demo.ts

```diff
@@ -34,13 +35,16 @@ export const getPatientByDemo = async ({
   if (!normalizedPatientDemo) return undefined;
 
   // Find patients based on the criteria of matching dob and genderAtBirth
-  const foundPatients = await patientLoader.findBySimilarity({
-    cxId,
-    data: {
-      dob: normalizedPatientDemo.dob,
-      genderAtBirth: normalizedPatientDemo.genderAtBirth,
+  const foundPatients = await patientLoader.findBySimilarity(
+    {
+      cxId,
+      data: {
+        dob: normalizedPatientDemo.dob,
+        genderAtBirth: normalizedPatientDemo.genderAtBirth,
+      },
     },
-  });
+    [["created_at", "ASC"]]
```

Comment:
whats the logic here? If we are gonna select the first patient returned, that will now be the first patient created. Not saying ASC vs DESC is right or wrong, just interested in the logic :)

Reply:
The idea is that without any sorting, the response from `getPatientByDemo` could be inconsistent in the case of multiple matches. By sorting we ensure that, in the event of multiple matches, we are always returning the first created (and that the response for the same demo data doesn't change overtime).

The decision to go with earliest created is arbitrary though. Another selection would be fine, it just might mean that overtime the response from this function changes (which opens up a new can of worms from a UX perspective).

Reply:
ok sounds good. I think the argument for latest over earliest is that we are returning the most recent information which you could argue is most likely to be the right match since its the last time a customer inputted information. Maybe just something to remember once this all gets revisited and levelled up down the line

Reply:
It's a good point. Personally I think we should hard fail on multiple matches or return undefined as `getPatientByDemo` simply isn't well defined in these cases if it's *supposed* to return a single patient. But, I think this is a larger discussion.

If you feel strongly, we can do latest, or I can bring it up to the larger team.